worker/include/RTC/IceServer.hpp

```diff
@@ -116,10 +130,30 @@ namespace RTC
 		 */
 		RTC::TransportTuple* HasTuple(const RTC::TransportTuple* tuple) const;
 		/**
-		 * Set the given tuple as the selected tuple.
+		 * Set the given tuple as the selected tuple. Returns true if given tuple
+		 * was not already the selected tuple, false otherwise.
 		 * NOTE: The given tuple MUST be already stored within the list.
 		 */
-		void SetSelectedTuple(RTC::TransportTuple* storedTuple);
+		bool SetSelectedTuple(RTC::TransportTuple* storedTuple);
+		bool AreConsentChecksEnabled() const
```

Comment:
Why plural? To me `consent check` represents the feature, even if it involves doing periodical checks.

If we renamed to `keep alive` it would be clear: `AreKeepAlivesEnabled` vs `IsKeepAliveEnabled`. IMHO the later makes more sense.

Reply:
I'm renaming to `IsConsentCheck`, etc. This is not a keep alive but ICE consent, which also involves checking the received STUN response from the client. If it's 403 then we must also disconnect (not done yet). Will address the other things later.

Reply:
> This is not a keep alive but ICE consent, which also involves checking the received STUN response from the client. If it's 403 then we must also disconnect (not done yet). Will address the other things later.

Yes, I know I know :+1: